Silicon/TigerlakePkg/Library/TxtInit/TxtLib.inf

```diff
@@ -0,0 +1,55 @@
+#/*++
```

Comment:
move library location from "Library/TxtInit/TxtLib.inf" to "Library/TxtLib/TxtLib.inf"